evohomeclient2/controlsystem.py

```diff
@@ -112,38 +115,46 @@ def temperatures(self):
             yield zone_info
 
     def zone_schedules_backup(self, filename):
-        """Backs up all zones on control system to the given file"""
-        print("Backing up zone schedule to: %s" % (filename))
+        """Backup all zones on control system to the given file."""
+        _LOGGER.setLevel(logging.INFO)
+        _LOGGER.info("Backing up schedules from ControlSystem: %s (%s)...",
+                     self.systemId, self.location.name)
 
         schedules = {}
 
         if self.hotwater:
-            print("Retrieving DHW schedule: %s" % self.hotwater.zoneId)
+            _LOGGER.info("Retrieving DHW schedule: %s...",
+                         self.hotwater.zoneId)
 
             schedule = self.hotwater.schedule()
             schedules[self.hotwater.zoneId] = {
-                'name': 'Domestic Hot Water', 'schedule': schedule}
+                'name': 'Domestic Hot Water',
+                'schedule': schedule}
 
         for zone in self._zones:
             zone_id = zone.zoneId
             name = zone.name
 
-            print("Retrieving zone schedule: %s - %s" % (zone_id, name))
+            _LOGGER.info("Retrieving Zone schedule: %s - %s", zone_id, name)
 
             schedule = zone.schedule()
             schedules[zone_id] = {'name': name, 'schedule': schedule}
 
         schedule_db = json.dumps(schedules, indent=4)
 
+        _LOGGER.info("Writing to backup file: %s...", filename)
         with open(filename, 'w') as file_output:
             file_output.write(schedule_db)
 
-        print("Backed up zone schedule to: %s" % filename)
+        _LOGGER.info("Backup completed.")
 
     def zone_schedules_restore(self, filename):
-        """Restores all zones on control system from the given file"""
-        print("Restoring zone schedules from: %s" % filename)
+        """Restore all zones on control system from the given file."""
+        _LOGGER.setLevel(logging.INFO)
+        _LOGGER.warning("Restoring schedules to ControlSystem %s (%s)...",
```

Comment:
This feels like it should still be at info as it's not warning about a failure somewhere

Reply:
I put it as a warning becuase it was making a change, but after reading (https://docs.python.org/3/howto/logging.html), I think you're right - should be `logging.INFO`.